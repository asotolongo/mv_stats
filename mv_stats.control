#extension
comment = 'Extension to track statistics about materialized view in the database'
default_version = '0.3.0'
superuser = true
