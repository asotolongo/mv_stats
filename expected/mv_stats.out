\set ECHO none
    mv_name    | refresh_count 
---------------+---------------
 public.mv_act |             1
 public.mv_bgw |             1
 public.mv_dat |             1
(3 rows)

 mv_activity_reset_stats 
-------------------------
 public.mv_act
(1 row)

    mv_name    | refresh_count 
---------------+---------------
 public.mv_bgw |             1
 public.mv_dat |             1
 public.mv_act |             0
(3 rows)

 mv_activity_reset_stats 
-------------------------
 public.mv_bgw
 public.mv_dat
 public.mv_act
(3 rows)

    mv_name    | refresh_count 
---------------+---------------
 public.mv_bgw |             0
 public.mv_dat |             0
 public.mv_act |             0
(3 rows)

